src/api/queries/gcpQuery.ts

```diff
@@ -0,0 +1,42 @@
+import * as utils from './query';
+
+export interface GcpFilters extends utils.Filters {
+  account?: string | number;
+}
+
+type GcpGroupByValue = string | string[];
+
+interface GcpGroupBys {
+  service?: GcpGroupByValue;
+  account?: GcpGroupByValue;
+  instance_type?: GcpGroupByValue;
+  org_unit_id?: GcpGroupByValue;
```

Comment:
You can remove anything with an "org" prefix

Reply:
Done